### DISCUSSION TITLE
Bienvenido a discusiones

### DISCUSSION THREAD

**Discussion by Paco8** [Paco8/plugin.video.movistarplus#10] [Announcements]
## 👋 ¡Bienvenido!

Usamos las discusiones como un lugar para conectar con otros miembros de nuestra comunidad. Esperamos que:

- Hagas preguntas sobre las que tengas curiosidad.
- Compartas ideas.
- Interactúes con otros miembros de la comunidad.
- Des la bienvenida a los demás y tengas una mente abierta. Recuerda que esta es una comunidad que construimos juntos 💪.

**Comment by Alfomso51** [comment]
Hola un saludo y mil gacias por este fantastico addon, tengo una consulta, lo tengo instalado en una raspberry pi 4b con libreelec y la tv la primera vez que la pongo se congela la imagen detengo la reproduccion y la pongo de nuevo y entonces va bien aunque con micro cortes e de decir que la pi la tengo por wifi por que no tengo ninguna posibilidad de ponerla por etherney

**Comment by dcbsant** [comment]
Hola. Gracias por el curro. Es una maravilla.

**Comment by Tarraco1** [comment]
Hola, buenos días en las actualizaciones del plugin de movistar en las anteriores no salía la marca de agua con un código en esta nueva si sale eso nonse podría eliminar? Gracias por tu trabajo eres un crack.!!

**Reply by Paco8** [reply]
Ya respondido aquí: https://github.com/Paco8/plugin.video.movistarplus/issues/40

**Comment by xxeku22** [comment]
Alguna manera de reproducción para este addons en un AppleTV

**Comment by Tarraco1** [comment]
Buenas tardes por que cuando estas un tiempo sin usar la cuenta de movistar se cierra la sesión y ahí que volver a iniciar sesión habría alguna forma que la sesión no se cerrarse y estuviese activa y que no se cierre gracias.

**Reply by Paco8** [reply]
El token movistar expira creo que a los 10 días, y si no lo usas en ese tiempo deja de funcionar.
Sí que quizás se podría implementar que el addon hiciera login automático cuando eso ocurra.

**Reply by Tarraco1** [reply]
Hola, crees que entoces podrías hacerlo sino te importa @Paco8 yo te lo agradecería,

**Comment by Tarraco1** [comment]
Hola buenos días por que cuando me meto a la cuenta me dice que tengo que volver a registrar y no se me ve nada pero si estoy registrado y se ve el panel de canales pero cuando le doy para visualizar un canal me pone que tengo que volver a registrar, gracias espero me puedas ayudar

**Comment by Tarraco1** [comment]
Te digo Paco exactamente lo que dice el mensaje de error:
Error de reproducción: Uno o más elementos no pudieron reproducirse. Consulte el registro para obtener más información acerca de este mensaje.

**Comment by gttres** [comment]
Buenos dias, a mí, una vez instalado aparentemente funcional, al pulsar un programa para visualizarlo me aparece una ventana que pone "Widevine CDM EULA" y un texto como diciendo algo sobre derechos de autor, con 2 opciones ACEPTAR o CANCELAR, si pulso CANCELAR no se reproduce nada y si pulso ACEPTAR a partir de ahí me sale error en kodi y no me reproduce nada. Todo esto en un kodi instalado en un pc.
Gracias

**Comment by Paco8** [comment]
Tienes que aceptar la instalación de widevine. ¿Que error sale después?

**Comment by gttres** [comment]
Hola, sale ventana típica de kodi: Error De Reproducción, Uno o más elementos no pudieron reproducirse, ....

**Comment by Paco8** [comment]
Mira en la configuración del addon si está activada la opción de usar el proxy para la licencia

**Comment by gttres** [comment]
Si, estaba activada, la desactivo y ahora me aparece ventana "Reproducción de lista abortada"

**Comment by gttres** [comment]
Además, haciéndolo desde TV y desde el propio ADDONS

**Comment by Paco8** [comment]
¿Qué versión de kodi tienes? Hace falta kodi 21 por lo menos, con una versión reciente de inputstream adaptive.

**Reply by gttres** [reply]
he probado con la 20 y 21, veré que versión tengo de inputstream y pruebo

**Comment by gttres** [comment]
La versión es la 21.5.23 y está en auto actualizar

**Comment by Paco8** [comment]
¿Te pasa en todos los canales?

**Reply by gttres** [reply]
SI

**Comment by Paco8** [comment]
Activa el registro en la configuración de kodi y luego mándamelo después de intentar ver un canal.

**Reply by gttres** [reply]
Hola, he reinstalado todo y no lo consigo, desisto, gracias por todo.
